Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -43,7 +65,8 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
 
         tableView.registerCell(BasicCell.self)
         tableView.registerCell(BorderedInnerTableCell.self)
-
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "Spacer")
```

Comment:
If you used the spacer method defined in TableView+Cells this line might need a change, please check how the method used exactly

Reply:
It is a different method. And I could not find a way to unify this.